cosmo_tester/test_suites/bootstrap_based_tests/inplace_upgrade_test.py

```diff
@@ -65,7 +65,7 @@ def test_inplace_upgrade(cfy,
                    interval=1)
     cfy.snapshots.download([snapshot_name, '-o', snapshot_path])
     cfy.teardown(['-f', '--ignore-deployments'])
-    cluster._bootstrap_manager()
```

Comment:
this was wrong before, because `_bootstrap_manager` used to take an inputs file argument

Reply:
Only since a few days ago ;)